rero_ils/modules/patrons/api.py

```diff
@@ -44,11 +38,30 @@
 from ..providers import Provider
 from ..utils import extracted_data_from_ref, get_patron_from_arguments, \
     get_ref_for_pid, trim_barcode_for_record
+from ...utils import create_user_from_data
 
 _datastore = LocalProxy(lambda: current_app.extensions['security'].datastore)
 
 current_patron = LocalProxy(lambda: Patron.get_patron_by_user(current_user))
 
+
+def create_patron_from_data(
+    data, delete_pid=False, dbcommit=False, reindex=False):
+    """Create a patron and a user from a data dict.
+
+    :param data - dictionary representing a library user
+    :param dbcommit - commit the changes in the db after the creation
+    :param reindex - index the record after the creation
+    :returns: - A Patron instance.
```

Comment:
I'm not aware of the docstring format, but I'm wondering why the last one is a sentence (capital and dot) and note the three precedent lines.